LottieSample/src/main/res/layout/fragment_list.xml

```diff
@@ -17,7 +17,7 @@
             android:layout_height="wrap_content"
             android:layout_gravity="center"
             app:lottie_fileName="Logo/LogoSmall.json"
-            app:lottie_loop="true" />
+            app:lottie_repeatCount="-1" />
```

Comment:
@fabionuno Hmm, this API is nice and consistent with the code now but it doesn't feel as nice to use. I'd be fine with leaving loop=true in (as you left in view_holder_grid below) as a convenience for this.